Objects/Converters/ConverterRevit/ConverterRevitShared/AllRevitCategories.cs

```diff
@@ -90,8 +90,14 @@ public IRevitCategoryInfo GetRevitCategoryInfo(Base @base)
         return categoryInfo;
       }
 
-      var instanceCategory = @base["category"] as string;
-      if (string.IsNullOrEmpty(instanceCategory)) return categoryInfo;
```

Comment:
Appreciate this one didn't change but recent coding standards request braces for all single line IF statements for readability. also clear line space between conditional blocks, even if the next line contains a comment

Reply:
Agreed, this is someone else's code I removed ;)